### ISSUE TITLE
L-o-S - change of data strategy = Deezer ONLY metadata/alldata source implementation 

### ISSUE THREAD

**Issue by Abraka** [youegraillot/lidarr-on-steroids#61]
Hello there, I would like to know your opinion on idea to make L-o-S deezer centric in way, that all info about music-metadata is downloaded from deezer and any other sources of data like musicbrainz and similar are discarded. Reason for this is constant mismatch of albums and songs downloaded from deezer and then compared with external musics dbs like musicbraninz.
Reason for running L-o-S is to get deezer artists, albums, songs. It should be then only about deezer datasources include musics metadata. It this sense I dont want or need comparison with others metadata sources because Im using deezer as a source and thus everything  should be deezer sourced.

Thx.

**Comment by nanometer5088** [comment]
Agreed. Musicbrainz data is very bad, specially for smaller artists.

**Comment by janaxhell** [comment]
I'm not sure if it can be done, but I strongly agree as well, but not as an exclusive. Best of both worlds would be default to Deezer metadata and IF album/artist is not present there, search Musicbrainz.

**Comment by Abraka** [comment]
Well, why not, that is a detail, we need D-O to be implemented ASAP. But its looks like L-o-S is nearly dead project as is deemix. Sad thing to watch :-1:

**Comment by codefaux** [comment]
L-o-S is honestly not capable of making this change, per se. The amount of reach they have within Lidarr and Deemix is very minimal.

I know this because I recently reverse-engineered it so its patched Deemix can be used with a standard plugins-enabled Lidarr installation, thus separating Deemix from Lidarr and enabling either to be updated separately as their parent projects mature.

This project (L-o-S) does three things.
- Installs a branch of Lidarr with plugins enabled
- Installs the Deemix plugin for Lidarr, and a small script addon
- Installs Deemix and applies a small patch
- Pre-sets user-configuration in both Lidarr and Deemix

There's no internal code change.

It doesn't rework the inside of Lidarr, it doesn't rework the inside of Deemix beyond one patch, it doesn't provide the linkage between Deemix and Lidarr's internals. Those are separate projects used by this one.

This project cannot pivot to provide metadata for Lidarr. This project is merely auto-installing and auto-configuring three other projects.

The creator of the Deemix-for-Lidarr plugin would be the person with whom to raise this request.
